pkg/lib/userexport/model_test.go

```diff
@@ -26,7 +26,7 @@ func TestRequest(t *testing.T) {
 		serialized, err := json.Marshal(request)
 		So(err, ShouldBeNil)
 
-		So(string(serialized), ShouldEqualJSON, requestBody)
+		So(string(serialized), ShouldEqualJSON, `{"csv":{"fields":null},"format":"ndjson"}`)
```

Comment:
This test is good. It shows a bug. If we serialize the request, then `"csv"` should be omitted if it is unspecified originally.

Reply:
added omitempty to csv field